### PR TITLE
Update 6-5-tostrud-climbing-wall.yaml

### LINKED ISSUE #1748: Building hours update for Tostrud Climbing Wall
## Before:

```yaml
name: Tostrud Climbing Wall
category: Gym

schedule:
  - title: Hours
    hours:
      - {days: [Mo, Tu, We, Th], from: '6:30am', to: '10:45pm'}
      - {days: [Fr], from: '6:30am', to: '8:45pm'}
      - {days: [Sa], from: '9:00am', to: '8:45pm'}
      - {days: [Su], from: '12:00pm', to: '10:45pm'}

breakSchedule:
  fall: []
  thanksgiving: []
  winter: []
  interim: []
  spring: []
  easter: []
  summer:
    - title: Hours
      hours: [{days: [Mo, Th], from: '4:00pm', to: '6:00pm'}]
```

## After:

```yaml
name: Tostrud Climbing Wall
category: Gym

schedule:
  - title: Hours
    hours:
      - {days: [Mo, Tu, We, Th], from: '6:30am', to: '10:45pm'}
      - {days: [Fr], from: '6:30am', to: '8:45pm'}
      - {days: [Sa], from: '9:00am', to: '8:45pm'}
      - {days: [Su], from: '12:00pm', to: '10:45pm'}
    notes: Sun - Fri 4-9 PM.    Sat - 1-5 PM

  - title: Hours
    hours:
      - {days: [Mo, Tu, We, Th, Fr, Su], from: '4:00pm', to: '9:00pm'}
      - {days: [Sa], from: '1:00pm', to: '5:00pm'}

breakSchedule:
  fall: []
  thanksgiving: []
  winter: []
  interim: []
  spring: []
  easter: []
  summer:
    - title: Hours
      hours: [{days: [Mo, Th], from: '4:00pm', to: '6:00pm'}]
```

### PR DESCRIPTION
**Pull request by hawkrives**
Closes #1748

### CODE REVIEW

File: data/building-hours/6-5-tostrud-climbing-wall.yaml [hunk 1]
```diff
@@ -4,10 +4,9 @@ category: Gym
 schedule:
   - title: Hours
     hours:
-      - {days: [Mo, Tu, We, Th], from: '6:30am', to: '10:45pm'}
-      - {days: [Fr], from: '6:30am', to: '8:45pm'}
-      - {days: [Sa], from: '9:00am', to: '8:45pm'}
-      - {days: [Su], from: '12:00pm', to: '10:45pm'}
+      - {days: [Mo, Tu, We, Th, Fr], from: '4:00pm', to: '9:00pm'}
```
Comment (rye) [inline data/building-hours/6-5-tostrud-climbing-wall.yaml]: This needs to be
- Sunday–Thursday: 4pm–9pm
- Friday: 4pm–8pm

**Review by rye — CHANGES_REQUESTED** [review]

File: data/building-hours/6-5-tostrud-climbing-wall.yaml (hunk 1 above)
Comment (hawkrives) [reply data/building-hours/6-5-tostrud-climbing-wall.yaml]: It is

Comment (hawkrives) [reply data/building-hours/6-5-tostrud-climbing-wall.yaml]: We have an unofficial rule of thumb that the schedules start on Monday

**Review by rye — APPROVED** [review]

File: data/building-hours/6-5-tostrud-climbing-wall.yaml (hunk 1 above)
Comment (hawkrives) [reply data/building-hours/6-5-tostrud-climbing-wall.yaml]: So I split his into M-Fr, Sa, Su

**Pull request merged**